net/databag/files/databag.init

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh /etc/rc.common
+ 
+START=10
+STOP=15
+
+config_databag() {
+	config_get store_path "$1" store_path "/tmp/store"
```

Comment:
use more unique foleder name, like `/tmp/databag/store`, and it should keep consistent with `option store_path`